app/src/main/java/com/appunite/loudius/ui/pullrequests/PullRequestsScreen.kt

```diff
@@ -122,46 +120,43 @@ private fun PullRequestsList(
 
 @Composable
 private fun PullRequestItem(
+    index: Int,
     data: PullRequest,
-    darkBackground: Boolean,
     onClick: (PulLRequestsAction) -> Unit,
 ) {
-    val backgroundColor = if (darkBackground) {
-        MaterialTheme.colorScheme.onSurface.copy(0.08f)
-    } else {
-        MaterialTheme.colorScheme.surface
-    }
-    Row(
-        modifier = Modifier
-            .fillMaxWidth()
-            .background(backgroundColor)
-            .clickable { onClick(PulLRequestsAction.ItemClick(data.id)) },
-    ) {
-        PullRequestIcon()
-        RepoDetails(pullRequestTitle = data.title, repositoryName = data.fullRepositoryName)
-    }
-    Divider(color = MaterialTheme.colorScheme.outlineVariant)
+    LoudiusListItem(
+        index = index,
+        modifier = Modifier.clickable { onClick(PulLRequestsAction.ItemClick(data.id)) },
+        icon = { modifier -> PullRequestIcon(modifier) },
+        content = { modifier ->
+            RepoDetails(
+                modifier = modifier,
+                pullRequestTitle = data.title,
+                repositoryName = data.fullRepositoryName,
+            )
+        },
+    )
 }
 
 @Composable
-private fun PullRequestIcon() {
-    Image(
+private fun PullRequestIcon(modifier: Modifier) {
+    LoudiusListIcon(
+        modifier = modifier,
         painter = painterResource(id = R.drawable.ic_pull_request),
-        contentDescription = null,
-        modifier = Modifier
-            .padding(start = 18.dp, top = 10.dp)
-            .size(width = 18.dp, height = 20.dp),
+        contentDescription = stringResource(id = R.string.pull_requests_screen_pull_request_content_description),
```

Comment:
**Question:** Is "Pull Requests" the correct content description for an icon? 
Also, the string name probably could be shorter - like `pull_request_icon_content_description`.

Reply:
I've added a task for this: https://loudius.atlassian.net/browse/LD-82